.github/labeler-issue.yml

```diff
@@ -1,14 +1,14 @@
-needs-triage:
+"status: needs triage":
   - '/.*/'
-api:
+"scope: api":
   - '(Devopness API)'
-website:
+"scope: website":
   - '(Devopness website)'
-web-app:
+"scope: web-app":
   - '(Devopness web app)'
-docs:
+"scope: docs":
   - '(Documentation)'
-sdk:
+"scope: sdk":
   - 'SDK-'
-sdk-javascript:
+"scope: sdk-javascript":
```

Comment:
@brlipi perhaps keep labels sorted alphabetically for easier maintenance and readability?
What do you think?

Reply:
Labels rearranged.